### PR TITLE
Fixes #4. Update expected return code to 202

### LINKED ISSUE #4: Throwingan unhandled exception with logstash 1.5.2
I am trying to submit some events to datadog from logstash. Below is my logstash config:-

```
input {
 stdin{ 
 type => "stdin-type" 
 codec => json {}
 }
}

output { 
  if "_jsonparsefailure" in [tags] {
    stdout { codec => rubydebug }
    datadog {
      api_key => "xxxxxx"      
    }
  }  
}
```

On typing some non-json input (let's say `hello`) in STDIN of logstash I am getting the below output and not seeing any events in datadog.

```
Logstash startup completed
hello
{
       "message" => "hello",
          "tags" => [
        [0] "_jsonparsefailure"
    ],
      "@version" => "1",
    "@timestamp" => "2015-09-05T16:55:41.652Z",
          "type" => "stdin-type",
          "host" => "jabong-1383"
}
Unhandled exception {:request=>"#<Net::HTTP::Post POST>", :response=>"#<Net::HTTPAccepted 202 Accepted readbody=true>", :exception=>"RuntimeError", :level=>:warn}
```

Logstash Version 1.5.2

### PR DESCRIPTION
**Pull request by llamashoes**
Thanks for contributing to Logstash! If you haven't already signed our CLA, here's a handy link: https://www.elastic.co/contributor-agreement/

### CODE REVIEW

File: lib/logstash/outputs/datadog.rb [hunk 1]
```diff
@@ -82,7 +82,7 @@ def receive(event)
       request.add_field("Content-Type", 'application/json')
       response = @client.request(request)
       @logger.info("DD convo", :request => request.inspect, :response => response.inspect)
-      raise unless response.code == '200'
+      raise unless response.code == '202'
```
Comment (jsvd) [inline lib/logstash/outputs/datadog.rb]: maybe we should support any 2xx response here with:

```ruby
raise unless response.code ~= /^2\d\d$/
```

Comment (llamashoes) [reply lib/logstash/outputs/datadog.rb]: That sounds like a more future-proof approach

**Comment by llamashoes** [comment]
Added another commit with the regex change.

**Review by jsvd — APPROVED** [review]
LGTM

**Pull request merged**